pythonwhat/tasks.py

```diff
@@ -447,7 +447,7 @@ def __init__(self, **kwargs):
     def __call__(self, shell):
         try:
             with capture_output() as out:
-                get_env(shell.user_ns)[self.fun_name](*self.arguments)
+                get_env(shell.user_ns)[self.fun_name](*self.arguments['args'], **self.arguments['kwargs'])
```

Comment:
The `*self.arguments['args']` refers to the dictionary element with key `'args'` that the student has to specify in `results` or `outputs` or `errors`.